fixtures/hdfs/core-site.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+<!-- Put site-specific property overrides in this file. -->
+
+<configuration>
+<property>
+  <name>fs.AbstractFileSystem.gs.impl</name>
+  <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS</value>
+  <description>The AbstractFileSystem for 'gs:' URIs.</description>
+</property>
+<property>
+  <name>fs.gs.project.id</name>
+  <value></value>
+  <description>
+    Optional. Google Cloud Project ID with access to GCS buckets.
+    Required only for list buckets and create bucket operations.
+  </description>
+</property>
+<property>
+  <name>google.cloud.auth.type</name>
+  <value>SERVICE_ACCOUNT_JSON_KEYFILE</value>
```

Comment:
How can we configure them via env?

Reply:
Probably not yet, looks like we need to update the `[hdrs](https://docs.rs/hdrs/latest/hdrs/)` library.

I learned that if running hdfs by Java, one can use `HADOOP_OPTS` to override configurations. For hadoop shell commands we can do similar things as `./bin/hadoop fs -D {property_name}={property_value}`. But as I know hdrs uses hadoop C API, and I haven't found any info about how to configure values via env, and let hadoop C library pick up the values.

So one possible way might be update `hdrs` to read certain environment variables and set them by calling [`hdfsBuilderConfSetStr`](https://github.com/Xuanwo/hdfs-sys/blob/d24061170aae6b8a96f5695b30e510d713d6e244/libhdfs/hdfs_3_3/include/hdfs/hdfs.h#L360) interface from [hdfs-sys](https://github.com/Xuanwo/hdfs-sys). Not sure if this works, but for this test cases' sake we can probably keep it this way.